tests/integration/test_commands.py

```diff
@@ -1,20 +1,1289 @@
+            "tests/data/example.Patient.manifest.csv"
+        )
+
+        # manifests have expected columns
+        assert list(biospecimen_df.columns) == [
+            "Sample ID",
+            "Patient ID",
+            "Tissue Status",
+            "Component",
+        ]
+        assert list(patient_df.columns) == [
+            "Patient ID",
+            "Sex",
+            "Year of Birth",
+            "Diagnosis",
+            "Component",
+            "Cancer Type",
+            "Family History",
+        ]
+        # manifests only have one row
+        assert len(biospecimen_df.index) == 1
+        assert len(patient_df.index) == 1
+        # manifests are empty except for component column which contains the name of the component
+        assert biospecimen_df["Component"].to_list() == ["Biospecimen"]
+        assert patient_df["Component"].to_list() == ["Patient"]
+        for column in ["Sample ID", "Patient ID", "Tissue Status"]:
+            assert np.isnan(biospecimen_df[column].to_list()[0])
+        for column in [
+            "Patient ID",
+            "Sex",
+            "Year of Birth",
+            "Diagnosis",
+            "Cancer Type",
+            "Family History",
+        ]:
+            assert np.isnan(patient_df[column].to_list()[0])
+
+    def test_generate_empty_google_sheet_manifests(
+        self,
+        runner: CliRunner,
+        testing_config: ConfigurationForTesting,
+    ) -> None:
+        """
+        Tests for:
+        - command has no errors, has exit code 0
+        - command output has file creation messages for 'Patient' and 'Biospecimen' manifest csvs
+        - command output has file creation messages for 'Patient' and 'Biospecimen' manifest links
+
+        both google sheets:
+        - have drop downs are populated correctly
+        - required fields are marked as “light blue”,
+        - non-required field are marked as white.
+        - first row comments are 'TBD'
+
+        Patient google sheet:
+        - first row of 'Family History has its own comment
+
+        Manual tests:
+        - Open 'CLI_TestManifestCommand_google_sheet_empty_patient.xlsx'
+        - Select 'Diagnosis' to be 'cancer' in the first row:
+            - 'Cancer Type' and 'Family History' cells in first row should be light blue.
+        - Select 'Diagnosis' to NOT be 'cancer' in the first row:
+            - 'Cancer Type' and 'Family History' cells in first row should be white
+
+        """
+        try:
+            result = runner.invoke(
+                manifest, ["--config", "config_example.yml", "get", "--sheet_url"]
+            )
+            # Assert these files were created:
+            assert os.path.isfile("tests/data/example.Biospecimen.manifest.csv")
+            assert os.path.isfile("tests/data/example.Patient.manifest.csv")
+            assert os.path.isfile("tests/data/example.Biospecimen.schema.json")
+            assert os.path.isfile("tests/data/example.Patient.schema.json")
```

Comment:
If the names of the files overlap with other tests this is likely to have issues when running the tests concurrently. Where one test is going to create files and remove them in places that might break other tests.

Is it possible to write these to a unique directory that applies to the specific test run, and/or get the cli runner to use a unique file name?

Reply:
So after lookign at the documentation, and some testing, the answer is kind of.

For the csvs: yes with `--output_csv`, but all csv files will be saved with this name, so the second one wil overwrite the first. This is ok for this test, since we are testing the google sheets.

Fopr the jsons: no

Reply:
So for the ones I could set a unique path for the output files I did so. For the others I put the removal of the file in an `if` block in case another test removes them first. I've also created this [issue](https://sagebionetworks.jira.com/browse/SCHEMATIC-209) that would allow for doing thsi in the future.